### PR TITLE
Edit all reference to github mozilla/kuma to mdn/kuma 

### PR DESCRIPTION
**Pull request by peterbe**
Part of #5612

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,16 +2,16 @@
 Kuma
 ====
 
-.. image:: https://travis-ci.org/mozilla/kuma.svg?branch=master
-   :target: https://travis-ci.org/mozilla/kuma
+.. image:: https://travis-ci.org/mdn/kuma.svg?branch=master
+   :target: https://travis-ci.org/mda/kuma
```
Comment (atopal) [inline README.rst]: typo?

File: README.rst [hunk 2]
```diff
@@ -53,14 +51,14 @@ Development
 .. _`P1 Bugs`: https://mzl.la/2BGZy44
 .. _`P2 Bugs`: https://mzl.la/2oniZa3
 .. _`All developer.mozilla.org bugs`: https://mzl.la/2onLvZ8
-.. _`Pull Request Queues`: http://prs.mozilla.io/mozilla:kuma%3Bmdn:kumascript,infra,mdn-fiori
-.. _`What's Deployed on MDN?`: https://whatsdeployed.io/s/HC0/mozilla/kuma
+.. _`Pull Request Queues`: http://prs.mozilla.io/mdn:kuma,kumascript,infra,mdn-fiori
```
Comment (tobinmori) [inline README.rst]: '%3Bmdn:' was replaced by a comma on line56. is that right?

File: docs/conf.py [hunk 3]
```diff
@@ -115,7 +115,7 @@
     'extra_nav_links': OrderedDict((
         ('MDN', 'https://developer.mozilla.org'),
         ('MDN Staging', 'https://developer.allizom.org'),
-        ('Kuma on GitHub', 'https://github.com/mozilla/kuma'),
+        ('Kuma on GitHub', 'https://github.com/mdn/kuma'),mdn
```
Comment (tobinmori) [inline docs/conf.py]: There's an extra 'mdn' appended on the change to 118?

Comment (peterbe) [reply docs/conf.py]: Pretty sad it slipped through without a pre-commit hook not catching it.

File: README.rst (hunk 2 above)
Comment (peterbe) [reply README.rst]: The line that remains works.

**Pull request merged**